docker/pravega/Dockerfile

```diff
@@ -9,6 +9,9 @@
 #
 FROM java:8
 
+RUN apt-get update && apt-get install -y -q rpcbind nfs-common \
```

Comment:
This is in the docker image itself. This is the usual way of doing updates/installs.